libs/server/Objects/Types/IGarnetObject.cs

```diff
@@ -35,8 +35,9 @@ public interface IGarnetObject : IDisposable
         /// <param name="output"></param>
         /// <param name="sizeChange"></param>
         /// <param name="removeKey"></param>
+        /// <param name="wrongType"></param>
         /// <returns></returns>
-        bool Operate(ref ObjectInput input, ref SpanByteAndMemory output, out long sizeChange, out bool removeKey);
+        bool Operate(ref ObjectInput input, ref SpanByteAndMemory output, out long sizeChange, out bool removeKey, out bool wrongType);
```

Comment:
Please clean this up if possible, remove size change and remove key. Put into enum perhaps?

Reply:
They go into GOSO as enum entries only if they are needed by the back end.